TelCo.ColorCoder/Colors.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{
+
+    internal static class Colors
```

Comment:
Why is the class static? is there a particular reason for making this class static? generally avoid static classes as they are difficult to maintain due to problems due to cached values